test/NNPDE_tests.jl

```diff
@@ -19,30 +26,45 @@ bcs = [u(0.,θ) ~ 1.0 , u(1.,θ) ~ 1.202]
 
 # Space and time domains
 domains = [t ∈ IntervalDomain(0.0,1.0)]
-
 # Discretization
-dx = 0.1
-discretization = NeuralPDE.PhysicsInformedNN(dx)
-
-# Neural network and optimizer
-opt = Flux.ADAM(0.1)
+dt = 0.1
+# Neural network
 chain = FastChain(FastDense(1,12,Flux.σ),FastDense(12,1))
 
+discretization = NeuralPDE.PhysicsInformedNN(dt,
+                                             chain,
+                                             init_params = nothing;
+                                             phi = nothing,
+                                             autodiff=false,
+                                             derivative = nothing,
+                                             training_strategies = NeuralPDE.TrainingStrategies(stochastic_loss=false))
+
 pde_system = PDESystem(eq,bcs,domains,[t],[u])
 prob = NeuralPDE.discretize(pde_system,discretization)
-alg = NeuralPDE.NNDE(chain,opt,autodiff=false)
-phi,res = solve(prob,alg,verbose=true, maxiters=1000)
+
+initθ = discretization.initθ
+p = DiffEqBase.NullParameters()
+loss = (θ) -> prob.f(θ,p)
+
+res1 = DiffEqFlux.sciml_train(loss, initθ, Flux.ADAM(0.1); cb = cb, maxiters=1000)
```

Comment:
Can you open an issue in GalacticOptim for Flux optimisers? 
I had read somewhere they planned on splitting it out to a separate package so we didn't add it in GalacticOptim yet.

Reply:
ok

Reply:
oh they weren't added? Just do it like how it was done in sciml_train. It's very straightforward.